Inventory.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using InventoryManagementSystem.ProductInfo;
+
+namespace InventoryManagementSystem.InventoryInfo
+{
+    public class Inventory
+    {
+        private readonly int _Id;
+        public static int InventoryIdCounter { get; private set; }
```

Comment:
What's the point of having this and `_id`?

Reply:
It's just an extra field for storing an id number for each product

Reply:
I don't think you need that, since you would know the id of the product from the product object itself. I don't see any benefit of storing id here. Also, I only see you incrementing this value from the constructor.